rosidl_typesupport_cpp/rosidl_typesupport_cpp/__init__.py

```diff
@@ -83,4 +91,18 @@ def generate_cpp(generator_arguments_file, type_supports):
                     template_file, data, generated_file,
                     minimum_timestamp=latest_target_timestamp)
 
+        elif extension == '.action':
+            spec = parse_action_file(pkg_name, ros_interface_file)
+            validate_field_types(spec, known_msg_types)
+            for template_file, generated_filename in mapping_actions.items():
+                generated_file = os.path.join(
+                    args['output_dir'], subfolder, generated_filename %
+                    convert_camel_case_to_lower_case_underscore(spec.action_name))
+
+                data = {'spec': spec, 'subfolder': subfolder, 'type_supports': type_supports}
```

Comment:
@apojomovsky same as above, are we using this `type_supports` list?

Reply:
it's not being used, just removed it. thanks!